src/pf/backend.lua

```diff
@@ -353,17 +353,30 @@ function emit_and_load(ssa, name)
    return func()
 end
 
+local function ast_to_ssa(ast)
+   local convert_anf = require('pf.anf').convert_anf
```

Comment:
I think this is a layering violation; probably don't want this here.  Given that pflua-quickcheck will be building internal AST data structures (and possibly SSA / ANF structures too) it makes more sense to me to have this in the quickcheck module.